lib/routes/hrbust/news.ts

```diff
@@ -0,0 +1,81 @@
+import { Route, ViewType } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const rootUrl = 'https://news.hrbust.edu.cn';
+
+export const route: Route = {
+    path: '/news/:category?',
+    name: '哈尔滨理工大学新闻网',
+    url: 'news.hrbust.edu.cn',
+    maintainers: ['cscnk52'],
+    handler,
+    example: '/hrbust/news',
+    parameters: { category: '栏目标识，默认为理工要闻' },
+    description: `
+    | lgyw | xwdd | zhenew | jxky | ycdt | xskc | jlhz | zsjy | djsz | zxbf | lgxb | mtlg | jzlt |
```

Comment:
Indent the content with either 0 or 2 spaces otherwise the markdown content won't be rendered properly.